### PR TITLE
Add path of unstarted containers to logs

### LINKED ISSUE #2857: vicadmin: not collecting logs for unstarted containers
vicadmin is not collecting logs for containers if output.log doesn't exist.
I assume the same is likely true for tether.debug.

This is problematic if the file you were after was vmware.log for a container that didn't get far enough for output. The resulting log bundle did not contain a directory for the container.

```
time=2016-10-22T03:05:12.120714213Z level=debug msg=[ END ] [main.listVMPaths:276] [36.680786ms]  
time="2016-10-22T03:05:12Z" level=info msg="Processed File read Path : ghicken-test_2/output.log" 
time="2016-10-22T03:05:12Z" level=info msg="Processed File write Path : ghicken-test/output.log" 
time="2016-10-22T03:05:12Z" level=info msg="Processed File read Path : ghicken-test_2/vmware.log" 
time="2016-10-22T03:05:12Z" level=info msg="Processed File write Path : ghicken-test/vmware.log" 
time="2016-10-22T03:05:12Z" level=info msg="Processed File read Path : ghicken-test_2/tether.debug" 
time="2016-10-22T03:05:12Z" level=info msg="Processed File write Path : ghicken-test/tether.debug" 
time="2016-10-22T03:05:12Z" level=info msg="Processed File read Path : elated_mclean-c73148967c7e0f5822697f5aeb971b5f1692a7450811cf4afd245e51dfcbfb7e/output.log" 
time="2016-10-22T03:05:12Z" level=info msg="Processed File write Path : elated_mclean-c73148967c7e0f5822697f5aeb971b5f1692a7450811cf4afd245e51dfcbfb7e/output.log" 
time="2016-10-22T03:05:12Z" level=info msg="Processed File read Path : elated_mclean-c73148967c7e0f5822697f5aeb971b5f1692a7450811cf4afd245e51dfcbfb7e/vmware.log" 
time="2016-10-22T03:05:12Z" level=info msg="Processed File write Path : elated_mclean-c73148967c7e0f5822697f5aeb971b5f1692a7450811cf4afd245e51dfcbfb7e/vmware.log" 
time="2016-10-22T03:05:12Z" level=info msg="Processed File read Path : elated_mclean-c73148967c7e0f5822697f5aeb971b5f1692a7450811cf4afd245e51dfcbfb7e/tether.debug" 
time="2016-10-22T03:05:12Z" level=info msg="Processed File write Path : elated_mclean-c73148967c7e0f5822697f5aeb971b5f1692a7450811cf4afd245e51dfcbfb7e/tether.debug" 
time=2016-10-22T03:05:12.136374856Z level=debug msg=[ END ] [main.findDatastoreLogs:326] [52.456518ms]  
time=2016-10-22T03:05:12.136621458Z level=debug msg=[BEGIN] [main.findDiagnosticLogs:171] 
time=2016-10-22T03:05:12.136721383Z level=debug msg=[ END ] [main.findDiagnosticLogs:171] [99.447Âµs]  
time=2016-10-22T03:05:12.136804535Z level=debug msg=[BEGIN] [main.(*server).bundleLogs:219] 
time=2016-10-22T03:05:12.136972118Z level=debug msg=[BEGIN] [main.zipEntries:266] 
time="2016-10-22T03:05:12Z" level=info msg="Collecting log with reader elated_mclean-c73148967c7e0f5822697f5aeb971b5f1692a7450811cf4afd245e51dfcbfb7e/output.log(main.datastoreReader{ds:(*object.Datastore)(0xc4204e1a40), path:\"elated_mclean-c73148967c7e0f5822697f5aeb971b5f1692a7450811cf4afd245e51dfcbfb7e/output.log\"})" 
time=2016-10-22T03:05:12.137170275Z level=debug msg=[BEGIN] [main.datastoreReader.open:367] elated_mclean-c73148967c7e0f5822697f5aeb971b5f1692a7450811cf4afd245e51dfcbfb7e/output.log 
time=2016-10-22T03:05:12.204509996Z level=debug msg=[BEGIN] [main.httpEntry:231] elated_mclean-c73148967c7e0f5822697f5aeb971b5f1692a7450811cf4afd245e51dfcbfb7e/output.log 
time=2016-10-22T03:05:12.204896704Z level=debug msg=[ END ] [main.httpEntry:231] [387.994Âµs] elated_mclean-c73148967c7e0f5822697f5aeb971b5f1692a7450811cf4afd245e51dfcbfb7e/output.log 
time=2016-10-22T03:05:12.205045987Z level=debug msg=[ END ] [main.datastoreReader.open:367] [67.876475ms] elated_mclean-c73148967c7e0f5822697f5aeb971b5f1692a7450811cf4afd245e51dfcbfb7e/output.log 
time="2016-10-22T03:05:12Z" level=warning msg="error reading elated_mclean-c73148967c7e0f5822697f5aeb971b5f1692a7450811cf4afd245e51dfcbfb7e/output.log({%!s(*object.Datastore=&{{/ha-datacenter/datastore/datastore1 0xc420219680 {Datastore 56a2eeef-3cafae6e-2bd3-000c29b00df4}}}) elated_mclean-c73148967c7e0f5822697f5aeb971b5f1692a7450811cf4afd245e51dfcbfb7e/output.log}): 404 Not Found\n" 
time="2016-10-22T03:05:12Z" level=info msg="Collecting log with reader df(\"df\")" 
time=2016-10-22T03:05:12.205420370Z level=debug msg=[BEGIN] [main.commandReader.open:178] df 
time=2016-10-22T03:05:12.206969680Z level=debug msg=[ END ] [main.commandReader.open:178] [1.539924ms] df 
time="2016-10-22T03:05:12Z" level=info msg="df has size 381" 
```

### LINKED ISSUE #3272: vic admin ui datastore value should be left justified
**VIC version:**

v0.8.0-rc1

In the image you can see the endpoint and system time are left justified. the datastore info stands out. 

screen shot 2016-11-21 at 4 44 30 pm

### PR DESCRIPTION
**Pull request by sgairo**
Fixes #2857 
Fixes #3272

### CODE REVIEW

File: cmd/vicadmin/logs.go [hunk 1]
```diff
@@ -328,7 +333,10 @@ func zipEntries(readers map[string]entryReader, out *zip.Writer) error {
 		// be explicit about the number of bytes to copy as the log files will likely
 		// be written to during this exercise
 		_, err = io.CopyN(w, e, sz)
-		_ = e.Close()
+		if e != nil {
+			_ = e.Close()
```
Comment (vburenin) [inline cmd/vicadmin/logs.go]: ```e.Close()``` is enough if you want to disregard an output.

File: cmd/vicadmin/logs.go [hunk 2]
```diff
@@ -328,7 +333,10 @@ func zipEntries(readers map[string]entryReader, out *zip.Writer) error {
 		// be explicit about the number of bytes to copy as the log files will likely
 		// be written to during this exercise
 		_, err = io.CopyN(w, e, sz)
```
Comment (vburenin) [inline cmd/vicadmin/logs.go]: Does it make sense to run a CopyN if e is nil?

File: cmd/vicadmin/logs.go (hunk 1 above)
Comment (sgairo) [inline cmd/vicadmin/logs.go]: Gotcha, thanks!

File: cmd/vicadmin/logs.go (hunk 2 above)
Comment (sgairo) [inline cmd/vicadmin/logs.go]: You're right, I adjusted it

File: cmd/vicadmin/logs.go [hunk 3]
```diff
@@ -273,8 +276,10 @@ func tarEntries(readers map[string]entryReader, out io.Writer) error {
 
 		// be explicit about the number of bytes to copy as the log files will likely
 		// be written to during this exercise
-		_, err = io.CopyN(t, e, e.Size())
-		_ = e.Close()
+		if e != nil {
+			e.Close()
+			_, err = io.CopyN(t, e, sz)
```
Comment (hickeng) [inline cmd/vicadmin/logs.go]: This is inverting the Close/Copy ordering from previously - is that intentional?

File: cmd/vicadmin/logs.go [hunk 4]
```diff
@@ -276,9 +276,9 @@ func tarEntries(readers map[string]entryReader, out io.Writer) error {
 
 		// be explicit about the number of bytes to copy as the log files will likely
 		// be written to during this exercise
-		_, err = io.CopyN(t, e, sz)
 		if e != nil {
 			e.Close()
+			_, err = io.CopyN(t, e, sz)
```
Comment (vburenin) [inline cmd/vicadmin/logs.go]: Are you sure you want to close e before you actually copy data?

Comment (sgairo) [inline cmd/vicadmin/logs.go]: Nope, will shift

File: cmd/vicadmin/logs.go (hunk 3 above)
Comment (sgairo) [inline cmd/vicadmin/logs.go]: Not intentional, will wheel around

File: cmd/vicadmin/logs.go (hunk 4 above)
Comment (vburenin) [inline cmd/vicadmin/logs.go]: Interesting, I didn't see George's comment while I was making mine even though he left it earlier.

File: cmd/vicadmin/logs.go [hunk 5]
```diff
@@ -300,9 +305,14 @@ func zipEntries(readers map[string]entryReader, out *zip.Writer) error {
 		e, err := r.open()
 		if err != nil {
 			log.Warningf("error reading %s(%s): %s\n", name, r, err)
-			continue
+
 		}
-		sz := e.Size()
+
+		sz := int64(0)
```
Comment (matthewavery) [inline cmd/vicadmin/logs.go]: this could also be `var sz int64` Looks a little neater.

File: cmd/vicadmin/logs.go [hunk 6]
```diff
@@ -253,12 +253,15 @@ func tarEntries(readers map[string]entryReader, out io.Writer) error {
 		e, err := r.open()
 		if err != nil {
 			log.Warningf("error reading %s(%s): %s\n", name, r, err)
-			continue
 		}
 
+		sz := int64(0)
```
Comment (matthewavery) [inline cmd/vicadmin/logs.go]: ditto

File: cmd/vicadmin/logs.go [hunk 7]
```diff
@@ -300,9 +305,14 @@ func zipEntries(readers map[string]entryReader, out *zip.Writer) error {
 		e, err := r.open()
 		if err != nil {
 			log.Warningf("error reading %s(%s): %s\n", name, r, err)
-			continue
+
```
Comment (matthewavery) [inline cmd/vicadmin/logs.go]: tiny cosmetic comment, removing this line will cut down on whitespace.

**Pull request closed without merging**